tests/e2e/rosa/main_test.go

```diff
@@ -50,3 +54,63 @@ func TestInstallOnlyParsesNodeProfiles(t *testing.T) {
 	}
 
 }
+
+func TestScanSetting(t *testing.T) {
+	f := framework.Global
+	// prinout all scan settings
+	scanSettingList := compv1alpha1.ScanSettingList{}
+	err := f.Client.List(context.TODO(), &scanSettingList)
+	if err != nil {
+		t.Fatalf("Failed to list scan settings: %v", err)
+	}
+	for _, scanSetting := range scanSettingList.Items {
+		t.Logf("ScanSetting: %s", scanSetting.Name)
+		for _, role := range scanSetting.Roles {
+			if role == "master" {
+				t.Fatalf("Role: %s is not expected", role)
+			}
+			t.Logf("Role: %s", role)
+		}
+	}
+
+	// List cluster claims
+	clusterClaimList := clusterv1alpha1.ClusterClaimList{}
+	err = f.Client.List(context.TODO(), &clusterClaimList)
+	if err != nil {
+		t.Fatalf("Failed to list cluster claims: %v", err)
+	}
+	for _, clusterClaim := range clusterClaimList.Items {
```

Comment:
I probably could have pulled this into it's own test in https://github.com/ComplianceAsCode/compliance-operator/pull/518.

We can clean this up after though.